<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBoxReceive">
     <property name="title">
      <string>接收区数据</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditReceive">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100000</width>
          <height>200</height>
         </size>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetReceiveButton" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClearReceive">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>清空接收区</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>串口配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>串口号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxPort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxBaud">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QComboBox" name="comboBoxData">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxStop">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxCheck">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>奇校验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>偶校验</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>接收区配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>接收模式</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxReceiveMode">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>HEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>文本</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>编码模式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxReceiveCodec">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>UTF-8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ASCII</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GBK</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBoxSend">
     <property name="title">
      <string>发送区数据</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditSend">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetSendButton" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSendOnTime">
           <property name="text">
            <string>定时发送/ms</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxSendOnTime">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSend">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClearSend">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>清空发送区</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>发送区配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>发送模式</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxSendMode">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>HEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>文本</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>编码模式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxSendCodec">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>UTF-8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ASCII</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GBK</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>串口链接</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonStartUSART">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonEndUSART">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>关闭串口</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelTime">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
